integration/spark/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -49,7 +49,7 @@ object CommonUtil {
           "Column group doesn't support Timestamp datatype:" + x)
       }
       // if invalid column is present
-      else if (dims.filter { dim => dim.column.equalsIgnoreCase(x) }.isEmpty) {
+      else if (!dims.exists { dim => dim.column.equalsIgnoreCase(x) }) {
```

Comment:
use `!dims.exists(_.column.equalsIgnoreCase(x))` instead of having `{` 
modify all places